inginious/common/tests/test_additional_field.py

```diff
@@ -0,0 +1,44 @@
+# pylint: disable=redefined-outer-name
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+import pytest
+
+from inginious.common.additional_field import AdditionalField
+
+
+@pytest.fixture()
+def init():
+    af = AdditionalField("test", "a description", 1)
```

Comment:
This is a little weird as it restricts testing to a very limited number of cases...while the goal of testing is actually to ensures the code works for a wide variety of cases.

Reply:
You're right but this will be complete within the dedicated test branch.